Dockerfile.dapper

```diff
@@ -4,6 +4,8 @@ ARG DAPPER_HOST_ARCH=amd64
 ARG http_proxy
 ARG https_proxy
 ENV HOST_ARCH=${DAPPER_HOST_ARCH} ARCH=${DAPPER_HOST_ARCH}
+ENV PROTOBUF_VER=3.7.1
+ENV PROTOBUF_S390_VER=3.18.0
```

Comment:
Why s390 use a different version?

Reply:
It is because protobuf v3.7.1 does not support s390.
Should we use the same version (3.18.0) for all protobuf utilities?

Reply:
As discussed, let's make the version consistent. Also need to check other repos as well.